server/kit/log.go

```diff
@@ -23,7 +24,7 @@ import (
 // If an empty string is provided, "gae_log" will be used in App Engine and "stdout" elsewhere.
 // For more information about to use of logID see the documentation here: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#FIELDS.log_name
 func NewLogger(ctx context.Context, logID string) (log.Logger, func() error, error) {
-	projectID, serviceID, svcVersion := getGAEInfo()
+	projectID, serviceID, svcVersion := observe.GetGAEInfo()
```

Comment:
The Stackdriver logger is also set up to be used outside of GAE. We should probably use the `GetServiceInfo` here.